src/main/resources/templates/application/partials/search-fragments.html

```diff
@@ -24,4 +24,33 @@ <h1 class="govuk-heading-l" th:text="${heading}">Default heading</h1>
       </div>
     </div>
   </form>
+</th:block>
+
+<th:block th:fragment="noResultsFragment">
+  <span class="govuk-caption-l" th:text="${caption}">Application search</span>
+  <h1 class="govuk-heading-l">
+    Application search - no results
```

Comment:
Should this include the ${caption} so its reusable?

Reply:
I had only been thinking of reuse as far as Application Search and Copy Case Search. 
Not sure it's doable beyond that, because the description text talks about creating a new application, and has a 'New Application' button